### PR TITLE
Brad funding post

### PR DESCRIPTION
**Pull request by bradmccarty**

### CODE REVIEW

File: astro/src/content/blog/fusionauth-funding.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+publish_date: 2023-10-31
```
Comment (mooreds) [inline astro/src/content/blog/fusionauth-funding.mdx:2]: You probably want this to be Nov 1, right?

File: astro/src/content/blog/fusionauth-funding.mdx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+Being a boostrapped company has always been a source of pride for us. However, it can also be a limiting factor. The investment from Updata Partners allows us to move faster than we would have been able to otherwise. As part of this investment round, two folks from Updata, Dan Moss, Principal, and Jon Seeber, General Partner, will join FusionAuth's board, bringing their expertise and insights to the company's journey.
+
+Founded in 2018, FusionAuth has achieved profitability and bootstrapped its way to success prior to this significant investment. The platform has already made its mark across six continents, serving over [450 customers](/customers-partners/?utm_source=blog&utm_medium=blog+post&utm_campaign=funding-announcement), including renowned brands such as Frontdoor, Oppenheimer & Co, Stihl, and Trusted Media Brands. We've grown by over 100% year over year, and we look forward to moving even faster now.
```
Comment (mooreds) [inline astro/src/content/blog/fusionauth-funding.mdx:15]: 2018?

File: astro/src/content/blog/fusionauth-funding.mdx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+What sets FusionAuth apart is our [free community edition](/download/?utm_source=blog&utm_medium=blog+post&utm_campaign=funding-announcement), which has been downloaded over 13 million times. This edition empowers developers to seamlessly integrate registration, login, and user management features into their applications while adhering to industry standards.
```
Comment (mooreds) [inline astro/src/content/blog/fusionauth-funding.mdx:17]: Are we tracking internal links? I thought I tried that in the past and it didn't work.

**Review by mooreds — APPROVED** [review]
Some style comments and a note about the date.

Nothing that blocks publishing it if the content is what you want.

File: astro/src/content/blog/fusionauth-funding.mdx (hunk 1 above)
Comment (bradmccarty) [reply astro/src/content/blog/fusionauth-funding.mdx:2]: Yeah. Thanks. I had to change to today to get it to appear in localhost.

File: astro/src/content/blog/fusionauth-funding.mdx (hunk 2 above)
Comment (bradmccarty) [reply astro/src/content/blog/fusionauth-funding.mdx:15]: According to Brian's timeline. That's when we took on the FA name.

**Comment by bradmccarty** [comment]
@mooreds any idea why my URLs keep failing? I don't have the absolute URLs anywhere that I can see. I've tried with a leading slash, and with no slash, it's still not working.

File: astro/src/content/blog/fusionauth-funding.mdx (hunk 3 above)
Comment (bradmccarty) [reply astro/src/content/blog/fusionauth-funding.mdx:17]: It *should* appear in Matomo. I hope?

**Comment by mooreds** [comment]
Looks like you figured it out :)

**Comment by bradmccarty** [comment]
> Looks like you figured it out :)

Nope. Still failing. Let me try adding the leading slash again and see if I just missed something.

**Comment by mooreds** [comment]
let me take a look.

**Comment by mooreds** [comment]
@bradmccarty all of the issues are due to the docs migration, not because of the blog post. You're fine to merge tomorrow.

**Pull request merged**